### ISSUE TITLE
Bug in masking tool

### ISSUE THREAD

**Issue by jo-moeller** [European-XFEL/EXtra-foam#292]
I use EXtra-foam/alpha and EXtra-geom to assemble AGIPD tiles
![Screenshot from 2021-03-18 14-47-44](https://user-images.githubusercontent.com/57397797/111638040-425e9280-87fa-11eb-8bb0-776f069e0de5.png)
When I then try to mask, I can't mask the lower part of the detector
![Screenshot from 2021-03-18 14-47-22](https://user-images.githubusercontent.com/57397797/111638187-63bf7e80-87fa-11eb-8b52-d538081c4e46.png)
This is as low as it gets. Everything below I can't draw a mask. I also get this output in the logger
![Screenshot from 2021-03-18 14-46-46](https://user-images.githubusercontent.com/57397797/111638367-8baee200-87fa-11eb-9bd0-16e4fd0d18fa.png)

**Comment by JamesWrigley** [comment]
Confirmed, the problem is that extra-geom doesn't have the [`output_array_for_dismantle_fast()`](https://foamalgo.readthedocs.io/en/latest/geometry.html#pyfoamalgo.geometry.AGIPD_1MGeometry.output_array_for_dismantle_fast) method mentioned in the error, which extra-foam's internal assembler does have. It will also need `dismantle_all_modules()`. It's breaking here: https://github.com/European-XFEL/EXtra-foam/blob/dev/extra_foam/pipeline/processors/image_processor.py#L281

I can't reproduce the issue with masking the lower part of the image though, does it work for you with the extra-foam assembler?

**Comment by jo-moeller** [comment]
True, it acutally doesn't depend on using EXtra-foam or EXtra-geom for assembling the tiles.
